newsroom/wire/search.py

```diff
@@ -11,9 +11,9 @@
 
 
 aggregations = {
-    'genre': {'terms': {'field': 'genre.name'}},
-    'service': {'terms': {'field': 'service.name'}},
-    'subject': {'terms': {'field': 'subject.name'}},
+    'genre': {'terms': {'field': 'genre.name', 'size': 10}},
```

Comment:
So we'll only list the top 10 genres, is that right?

Reply:
that's the default value, we should list them all